libs/elements/src/select/vl-select.element.ts

```diff
@@ -108,26 +116,24 @@ export class VlSelect extends vlFormValidationElement(BaseElementOfType(HTMLSele
         return 'data-vl-select-dressed';
     }
 
-    _successChangedCallback(oldValue: string, newValue: string) {
+    /**
+     *
+     * @param oldValue {String} - niet in gebruik
+     * @param newValue {String}
+     */
+    _successChangedCallback(_: string, newValue: string) {
         this.__stateChangedCallback(newValue, 'success');
     }
 
-    _errorChangedCallback(oldValue: string, newValue: string) {
+    /**
+     *
+     * @param oldValue {String} - niet in gebruik
+     * @param newValue {String}
+     */
+    _errorChangedCallback(_: string, newValue: string) {
```

Comment:
ook al is het niet in gebruik, behoud mss hier ook dezelfde benaming als we in andere oude files doen, dus `oldValue` ipv `_`

sowieso is  het een goed idee dat de JSDocs conform zijn wat de functie definiëert